ManatiCR/FDI
# Feature/fdi 26 recursos juridicos

[FDI-26](https://manati.atlassian.net/browse/FDI-26)
-
**Steps to test**
- Run `./scripts/fdi_local_install.sh` (to create the taxonomy terms)
- Log in as an admministrador de contenido user
- Create one node recurso jurídico, check you can edit it and delete it as well
